src/System.Private.ServiceModel/src/System/IdentityModel/Policy/UnconditionalPolicy.cs

```diff
@@ -87,22 +87,21 @@ private void Initialize(ClaimSet issuer, ClaimSet issuance, ReadOnlyCollection<C
             _issuances = issuances;
             _expirationTime = expirationTime;
 
-            // We don't yet support WindowsClaimSet
-            //if (issuance != null)
-            //{
-            //    _disposable = issuance is WindowsClaimSet;
-            //}
-            //else
-            //{
-            //    for (int i = 0; i < issuances.Count; ++i)
-            //    {
-            //        if (issuances[i] is WindowsClaimSet)
-            //        {
-            //            this.disposable = true;
-            //            break;
-            //        }
-            //    }
-            //}
+            if (issuance != null)
+            {
+                _disposable = issuance is WindowsClaimSet;
+            }
+            else
+            {
+                for (int i = 0; i < issuances.Count; ++i)
+                {
+                    if (issuances[i] is WindowsClaimSet)
+                    {
+                        _disposable = true;
+                        break;
+                    }
+                }
+            }
 
             _disposable = false;
```

Comment:
If the if block above sets _disposable to true, you are just going to revert it here again. I presume this was temporary code that should have been removed.